src/devinterp/slt/llc.py

```diff
@@ -115,19 +129,19 @@ def sample(self):
             "llc/means": self.llc_means.cpu().numpy(),
             "llc/stds": self.llc_stds.cpu().numpy(),
             "llc/trace": self.llcs.cpu().numpy(),
```

Comment:
since `self.moving_avg_llcs` is taking the place of `self.llcs`, do we want to return the former here or is the behavioral change w/ returning the latter intended?